src/Illuminate/Database/Schema/MySqlSchemaState.php

```diff
@@ -85,7 +85,7 @@ public function load($path)
      */
     protected function baseDumpCommand()
     {
-        $command = 'mysqldump '.$this->connectionString().' --no-tablespaces --skip-add-locks --skip-comments --skip-set-charset --tz-utc --column-statistics=0';
+        $command = 'mariadb-dump '.$this->connectionString().' --no-tablespaces --skip-add-locks --skip-comments --skip-set-charset --tz-utc --column-statistics=0';
```

Comment:
Wouldn't this break running command on MySQL databasw without `mariadb-dump` available?

Reply:
Yes you are right about that. 
```suggestion
        $command = 'mysqldump '.$this->connectionString().' --no-tablespaces --skip-add-locks --skip-comments --skip-set-charset --tz-utc --column-statistics=0';
```